src/lib/strategies/AbstractKeyEventStrategy.js

```diff
@@ -303,6 +338,27 @@ class AbstractKeyEventStrategy {
     return keyMapSummary;
   }
 
+  _createSequenceFromConfig(keyMapConfig) {
+    return arrayFrom(keyMapConfig).map((sequenceOrKeyMapOptions) => {
+      if (isObject(sequenceOrKeyMapOptions)) {
+        /**
+         * Support syntax:
+         * [
+         *   { sequence: 'a+b', action: 'keyup' },
+         *   { sequence: 'c' }
+         * ]
+         */
+        return copyAttributes(sequenceOrKeyMapOptions, {}, SEQUENCE_ATTRIBUTES);
+      } else {
+        /**
+         * Support syntax:
+         * 'a+b'
+         */
+        return { sequence: sequenceOrKeyMapOptions };
```

Comment:
⬜️ ⚔ Add the `action` attribute here, for uniformity of the data structure returned